src/Tests/InstrEngineTests/InstrEngineTests/InstrEngineTests.csproj

```diff
@@ -21,6 +21,7 @@
   </ItemGroup>
   <ItemGroup>
     <None Remove="EmbeddedResources\InvalidCSharp\readme.txt" />
+    <None Remove="EmbeddedResources\InvalidCSharp\RefStructsTestsImpl.il" />
```

Comment:
><None Remove="EmbeddedResources\InvalidCSharp\RefStructsTestsImpl.il" /> [](http://example.com/codeflow?start=4&length=72)

Why?

Reply:
I believe this was added automatically; I can remove it.